docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -2501,3 +2504,133 @@ function sitenow_update_9065() {
+      'alert-success' => 'alert--success',
+      'alert-info' => 'alert--info',
+    ];
+
+    $entity_type_manager = \Drupal::entityTypeManager();
+    $entity_field_manager = \Drupal::service('entity_field.manager');
+    $text_long = $entity_field_manager->getFieldMapByFieldType('text_long');
+    $text_with_summary = $entity_field_manager->getFieldMapByFieldType('text_with_summary');
+    $entity_field_map = array_merge($text_long, $text_with_summary);
+
+    foreach ($entity_field_map as $entity_type_id => $field_map) {
+      $entity_storage = $entity_type_manager->getStorage($entity_type_id);
+      if (!($entity_storage instanceof SqlContentEntityStorage)) {
+        continue;
+      }
+
+      $entity_type = $entity_type_manager->getDefinition($entity_type_id);
+      $field_storage_definitions = $entity_field_manager->getFieldStorageDefinitions($entity_type_id);
+      $table_mapping = $entity_storage->getTableMapping($field_storage_definitions);
+      foreach (array_intersect_key($field_storage_definitions, $field_map) as $field_storage_definition) {
+        $tables = [];
+        $field_name = $field_storage_definition->getName();
+        try {
+          $tables[] = $table_mapping->getFieldTableName($field_name);
+          $value_column = $table_mapping->getFieldColumnName($field_storage_definition, 'value');
+        }
+        catch (SqlContentEntityStorageException $e) {
+          continue;
+        }
+        // See if the field has a revision table.
+        $revision_table = NULL;
+        if ($entity_type->isRevisionable() && $field_storage_definition->isRevisionable()) {
+          if ($table_mapping->requiresDedicatedTableStorage($field_storage_definition)) {
+            $revision_table = $table_mapping->getDedicatedRevisionTableName($field_storage_definition);
+          }
+          elseif ($table_mapping->allowsSharedTableStorage($field_storage_definition)) {
+            $revision_table = $entity_type->getRevisionDataTable() ?: $entity_type->getRevisionTable();
+          }
+          $tables[] = $revision_table;
+        }
+        if (!empty($tables)) {
+          $db = \Drupal::database();
+
+          foreach ($tables as $table) {
+            $update_count = 0;
+            $query = $db->select($table, 't')
+              ->fields('t');
+
+            $or_group = $query->orConditionGroup();
+
+            foreach ($search_replace as $needle => $string) {
+              $or_group->condition($value_column, '%' . $db->escapeLike($needle) . '%', 'LIKE');
+            }
+
+            $results = $query->condition($or_group)
+              ->execute();
+
+            foreach ($results as $record) {
+
+              $document = Html::load($record->{$value_column});
+
+              foreach (['div'] as $tag) {
+                $instances = $document->getElementsByTagName($tag);
+
+                // We don't actually need the regressive loop
+                // here, but its copied/pasted.
```

Comment:
This comment is confusing since it does appear that the regressive loop is being used.